chapters/license-matching-guidelines-and-templates.md

```diff
@@ -30,9 +32,17 @@ Matched text should only include that found in the vetted license or exception t
 
 ### B.3.4 Guideline: replaceable text <a name="B.3.4"></a>
 
-Some licenses include text that refers to the specific copyright holder or author, yet the rest of the license is exactly the same as a generic version. The intent here is to avoid the inclusion of a specific name in one part of the license resulting in a non-match where the license is otherwise an exact match (e.g., the third clause and disclaimer in the BSD licenses, or the third, fourth, and fifth clauses of Apache-1.1). In these cases, there should be a positive license match.
-
-Text that can be considered replaceable for matching purposes is indicated in the SPDX License List template with mark-up and in the corresponding HTML pages with colored text. The text indicated as such can be replaced with similar values (e.g., a different name or generic term; different date) and still be considered a positive match. This rule also applies to text-matching in official license headers (see Guideline #1).
+Some licenses include text that refers to the specific copyright holder or author, yet the rest of the license is exactly the same. The intent here is to avoid the inclusion of a specific name in one part of the license resulting in a non-match where the license is otherwise an exact match to the legally substantive terms (e.g., the third clause and disclaimer in the BSD licenses, or the third, fourth, and fifth clauses of Apache-1.1). In these cases, there should be a positive license match.
+
+The text indicated as such can be replaced with similar values (e.g., a different name or generic term; different date) and still be considered a positive match. This rule also applies to text-matching in official license headers (see Guideline: official license headers).
+  
+The following XML element tag is used to implement this guideline.  `<alt>` with 2 attributes:
+* match: a POSIX extended regular expression (ERE) to match the replaceable text
+* name: An identifier for the variable text unique to the license XML document
+The original text is enclosed within the alt tags.
```

Comment:
... "within the beginning and ending tag of the `<alt>` element."

Reply:
if we say this here, then we have to say it everywhere else... I think the example is enough?

Reply:
@jlovejoy I agree with @zvr on this change - the `alt` tag uses the data between the tags, so adding that information here is important to clarify the use.  I looked at the other tag references and I think they are fine without having to add specifics.

Reply:
updated in next commit